src/libraries/ModuleEntityLib.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+//
+// See LICENSE-MIT file for more information
+
+pragma solidity ^0.8.20;
+
+import {ModuleEntity} from "@erc6900/reference-implementation/interfaces/IModularAccount.sol";
+
+library ModuleEntityLib {
+    // Magic value for hooks that should always revert.
+    ModuleEntity internal constant _PRE_HOOK_ALWAYS_DENY = ModuleEntity.wrap(bytes24(uint192(2)));
```

Comment:
We should re-copy this from the RI, because it's missing some parts like the byte layout diagram, and contains old fields like `_PRE_HOOK_ALWAYS_DENY`.

Reply:
Woops, good catch.

Reply:
Wait, I did copy this from the reference implementation 😂 . So I guess that needs to be cleaned up?

Reply:
oof, yeah we should clean that up. will get on it...

Reply:
https://github.com/erc6900/reference-implementation/pull/183